pkg/cmd/uninstall/uninstall.go

```diff
@@ -236,6 +236,14 @@ func (o *UninstallOptions) deleteNamespace(namespace string) error {
 // of an unrelated error
 func (o *UninstallOptions) DeleteReleaseIfPresent(ns string, releaseName string, errors []error, force bool) []error {
 	if err := o.Helm().StatusRelease(ns, releaseName); err == nil || force {
+
+		err = o.setPVCFinalizerToNull(ns, releaseName)
```

Comment:
Actually, out of interest, why do you call `setPVCFinalizerToNull` from here? The way `setPVCFinalizerToNull` is implemented, it is a global thing w/o any dependence to the release name.